src/components/Section.js

```diff
@@ -4,16 +4,21 @@ export default class Section {
     this._renderer = renderer;
     this._container = container;
   }
-  renderItems() {
-    this._items.forEach((data) => {
+  //lets cause some problems to undo remove first items and add "this._" to second items
+  renderItems(items) {
+    console.log("render items fired");
+    items.forEach((data) => {
       this._renderer(data);
     });
   }
 
   addItem(listElement) {
-    console.log("add item fired");
     this._container.prepend(listElement);
   }
+
+  addInitialItems(listElement) {
```

Comment:
Needs correction: there should not be such method, you should render the initial items using `renderItems` method

Reply:
This particular Method was added because of the order in which cards are loaded from the server vs posted when just adding a single card. one uses append and the other uses prepend. otherwise cards added show up in reverse order when page is refreshed

Reply:
Needs correction: If so then you should rename this method to something like `appendItem`, since the Section should not know anything about your logic